content/docs/command-reference/add.md

```diff
@@ -374,33 +376,34 @@ outs:
     path: data.xml
 ```
 
+[linked]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+[external cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
+
 ## Example: Transfer to remote storage
 
-When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later.
+When you want to add a large dataset that is outside of your
+<abbr>project</abbr> (e.g. online), you would normally need to download or copy
+it into the <abbr>workspace</abbr> first. But you may not have enough local
+storage space.
+
+Using the `--to-remote` flag lets you create a `.dvc` file for the target data
+without downloading it locally. DVC transfers it directly to [remote storage]
+instead. This way it can be [pulled](/doc/command-reference/plots) later, for
+example on a system that can handle it.
 
-Let's add the `data.xml` to our remote storage from the given remote location:
+Let's add a `data.xml` file via HTTP in this way. Note that the `--out` (`-o`)
+option is also needed since the target data is outside the project:
```

Comment:
As noted above, `--out` should be optional.

Reply:
Fixed, PTAL.